x-pack/qa/full-cluster-restart/src/test/java/org/elasticsearch/xpack/restart/FullClusterRestartIT.java

```diff
@@ -389,12 +412,16 @@ public void testApiKeySuperuser() throws IOException {
 
             // read is ok
             final Request searchRequest = new Request("GET", ".security/_search");
-            searchRequest.setOptions(
-                expectWarnings(
-                    "this request accesses system indices: [.security-7], but in a future major "
-                        + "version, direct access to system indices will be prevented by default"
-                ).toBuilder().addHeader("Authorization", apiKeyAuthHeader)
-            );
+            // Configure the warning to be optional due to #82837, it is ok since this test is for something else
```

Comment:
maybe add a TODO to change this back or as part of the fix for #82837  ? This comment can be enough on its own though :)

Reply:
Good point. I added a TODO. Thanks!